.github/workflows/gui-deploy.yaml

```diff
@@ -0,0 +1,32 @@
+name: Claim Gui Deployment
+
+on:
+  push:
+    branches: master
+    paths:
+      - claim-gui/**
+      - tooling/_harvest/checkpoints/**
+
+env:
+  REACT_APP_INFURA_KEY: ${{ secrets.INFURA_API_KEY  }}
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: Deploy Site
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
```

Comment:
other actions are using actions/setup-node@v3

Any significant differences between versions?

Reply:
hmm not sure ill upgrade it.